apps/cookbook/src/app/examples/menu-example/examples/advanced.ts

```diff
@@ -3,10 +3,15 @@ import { Component } from '@angular/core';
 const config = {
   selector: 'cookbook-menu-advanced-example',
   template: `<kirby-menu [closeOnSelect]="false">
-  <kirby-item (click)="actionClicked()">
+  <kirby-item [selectable]="true">
```

Comment:
kirby-item must be selectable when within a menu to make ion-item add a wrapping button